File: Scripting-Clase 1/Player.lua
require("Entity")

Player = {}
Player.__index = Player

function Player:New(num)
	local this =
	{
		base = Entity:New(num)
	}
	setmetatable(this,self)
	return this
end

function InitPlayer()
	InitSprite()
	SetPosition()
	SetSpeed()
end
function InitSprite()
	SetPlayerSprite("pixil-frame-0 (5).png")
end

function SetPosition()
	SetPlayerPosition(200,200)
end

function SetSpeed()
	PlayerSpeed(10)
end

File: Scripting-Clase 1/Player.cpp
 #include "pch.h"
#include "Player.h"

Player::Player(sf::Texture& _texture, const float x, const float y)
{
	InitSprite(_texture);
	//_sprite.setScale();
	SetPosition(x, y);
	this->posX = x;
	this->posY = y;
}

Player::~Player()
{

}

void Player::Update(const float& dt)
{

}